src/components/inline_edit/inline_edit_form.tsx

```diff
@@ -51,7 +60,19 @@ export type EuiInlineEditCommonProps = CommonProps & {
   /**
    * Props that will be applied directly to the EuiFieldText displayed in editMode
    */
-  editModeProps?: EuiFieldTextProps;
+  editModeProps?: {
+    formRowProps?: Partial<EuiFormRowProps>;
+    inputProps?: EuiFieldTextProps;
```

Comment:
I don't know if this _needs_ to be a partial, but I think adding the `Partial<>` notation makes the intention clear here
```suggestion
    inputProps?: Partial<EuiFieldTextProps>;
```